Cargo.toml

```
@@ -37,10 +37,10 @@ bytes = "1.3.0"
 clap = { version = "4.0.29", features = ["derive"] }
 console-subscriber = "0.1.8"
 ctrlc = "3.2.4"
-db = { package = "forest_db", git = "https://github.com/ChainSafe/forest", features = ["rocksdb"] }
+db = { package = "forest_db", git = "https://github.com/ChainSafe/forest", rev = "3e682310482718182cb2495fbaec5e575e5884e5", features = ["rocksdb"] }
 dirs = "4"
 dotenv = "0.15.0"
-ethers = { git = "https://github.com/gakonst/ethers-rs" }
+ethers = { git = "https://github.com/gakonst/ethers-rs", rev = "16f9fab75cb43e2d9e4c2b74b68afde7577f0e2f" }
```

```suggestion
ethers = "1.0.2"
```
Maybe we just do this. I think we were specifying the git before because we had it forked briefly